src/sql.rs

```diff
@@ -100,10 +97,8 @@ byond_fn! { sql_check_query(id) {
 // ----------------------------------------------------------------------------
 // Main connect and query implementation
 
-lazy_static! {
-    static ref POOL: RwLock<HashMap<String, Pool>> = Default::default();
-    static ref NEXT_ID: std::sync::atomic::AtomicUsize = Default::default();
-}
+static POOL: Lazy<DashMap<String, Pool>> = Lazy::new(DashMap::new);
+static NEXT_ID: AtomicUsize = AtomicUsize::new(0);
```

Comment:
No clue why an AtomicUsize was in a lazy_static. That was redundant and probably had a tiny performance penalty.